.github/workflows/scripts.yml

```
@@ -0,0 +1,29 @@
+name: CI with script
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install packages
+        run: sudo apt install -y --no-install-recommends pkg-config libgmp-dev curl ninja-build jq
```

```suggestion
        run: sudo apt install -y --no-install-recommends pkg-config curl
```
This should download and build gmp, so no need to install it. It also uses Make by default, so ninja shouldn't be needed either.

---

Resolved with resolve.